source/blueprints/hr_assistant.rst

```diff
@@ -0,0 +1,699 @@
+       - ``money``: detects whether the user is referring to salary information. For example: "What is {ivan|name} {earning|money}"
+       - ``dob``: detects whether the user is referring to date of birth. For example: "When was {Nan|name} {born|dob}?"
+       - ``employment_action``: detects whether the user is referring to hiring or firing an employee. For example: "What was {ivan|name}'s {date of termination|employment_action}"
+
+
+   - Compare/Functions
+       - ``comparator``: detects comparison keywords (more than, less than, equal to, between). For example: "is {mia|name} {earning|money} {more than} {ivan|name}?"
+       - ``extreme``: detects extreme keywords (highest, oldest, lowest, youngest). For example: "who is the {oldest|extreme} employee?"
+       - ``date_compare``: detects date comparision key words (prior to, after). For example: "Was {Ivan|name} {born|dob} {prior to|date_compare} {1990|sys_time}?"
+       - ``function``: detects a function type (percent, sum, average, count). For example: "What {percent|function} of employees are {women|sex}?"
+
+   - Custom Time Entities
+       - ``time_interval``: detects a decade (1980's, 80s, eighties). For example: "{how many|function} employees were {born|dob} in the {eighties|time_interval}?"
+       - ``time_recur``: detects a recurring time interval (yearly, monthly, weekly). For example: "what does {ivan|name} {make|money} {monthly|time_recur}?"
+
+
+HR assistant uses three system entities: ``sys_time`` (time), ``sys_amount-of-money`` (money), ``sys_number`` (number). Some examples for annotation with system entities: "{How many|function} employees were {born|dob} in {2009|sys_time}?" and "what {fraction|function} of employees {make|money} {less than|comparator} {69 grand|sys_amount-of-money}?".
+
+.. admonition:: Exercise
+
+To train the different machine learning models in the NLP pipeline for this app, we need labeled training data that covers all our intents and entities. To download the data and code required to run this blueprint, run the command below in a directory of your choice. (If you have already completed the Quick Start for this blueprint, you should skip this step.)
+
+.. code-block:: shell
+
+    python -c "import mindmeld as mm; mm.blueprint('hr_assistant');"
+
+This should create a MindMeld project folder called ``hr_assistant`` in your current directory with the following structure:
+
+.. image:: /images/hr_assistant_directory.png
+    :width: 250px
+    :align: center
+
+4. Dialogue States
+^^^^^^^^^^^^^^^^^^
+
+Dialogue state logic can be arbitrarily complex. Simple dialogue state handlers just return a canned text response, while sophisticated ones can call third party APIs, calculate state transitions, and return complex responses.
+
+MindMeld supports three ways to organize dialogue states in the Dialogue Manager:
+
```

Comment:
I agree this discussion seems moot. I dont understand the `Define **one dialogue state for multiple intents**` description since I dont see how one dialogue state is being used across multiple intents. I think we should just state there are various ways of matching to a dialogue state, including through intents and entities. @serapio What do you think?

Reply:
That's directly from the docs for the home assistant (https://www.mindmeld.com/docs/blueprints/home_assistant.html#dialogue-states)
For the other blueprints we are simply stating that we need one dialogue state for each intent, but on the home assistant there is discussion about ways 1 & 2. I think the impulse to add #3 was the recognition that the previous docs were over-simplifying things, but I think it would be better to just have the facts of the implementation here. The generalizations would fit better in Step 4, even though kwik-e-mart is just using one intent per dialogue state.

Reply:
Got it, yes I now understand the second point. I have removed this description from the docs.